tritium-caffeine/src/test/java/com/palantir/tritium/metrics/caffeine/CaffeineCacheMetricsTest.java

```diff
@@ -74,16 +81,48 @@ void estimatedSize() {
     }
 
     @Test
-    void maximumSize() {
+    void unboundedMaximumSize() {
+        when(cache.policy().eviction()).thenReturn(Optional.empty());
```

Comment:
working around https://github.com/mockito/mockito/issues/2865 difference between Mockito 4.x & 5.0 which leads to [test failure](https://app.circleci.com/pipelines/github/palantir/tritium/2642/workflows/773f100f-fe40-4d13-9e03-a2337249bc67/jobs/19318/tests)

```
java.lang.ClassCastException: class com.github.benmanes.caffeine.cache.Policy$Eviction$MockitoMock$nsH7a8Ju cannot be cast to class java.lang.Long (com.github.benmanes.caffeine.cache.Policy$Eviction$MockitoMock$nsH7a8Ju is in unnamed module of loader 'app'; java.lang.Long is in module java.base of loader 'bootstrap')
	at com.palantir.tritium.metrics.caffeine.CaffeineStats.lambda$weightedSize$1(CaffeineStats.java:60)
	at com.palantir.tritium.metrics.caffeine.CaffeineCacheMetricsTest.lambda$weightedSize$2(CaffeineCacheMetricsTest.java:87)
	at org.assertj.core.api.AbstractObjectAssert.returns(AbstractObjectAssert.java:1106)
	at com.palantir.tritium.metrics.caffeine.CaffeineCacheMetricsTest.weightedSize(CaffeineCacheMetricsTest.java:87)
```